README.md

```diff
@@ -128,6 +128,8 @@ Then, whenever the framework content is changed:
 - alternatively, run `npm run frontend-build:watch` to automatically rebuild the framework content whenever a framework YML file changes
 - generate schemas into the API's `json_schema` directory using the `scripts/generate-schemas.py` script in this repo.
 
+Alternatively, if you think `bower link` is more trouble than it's worth, you can just symlink the your project's `bower_components/digitalmarketplace-frameworks` to wherever this repo lives on your local filesystem and it's 99% as good as the above.
```

Comment:
What's the problem with `bower link`? I'd rather we document one common way so that we don't have to go through a checklist of "how did you set this up" when someone has an issue with their local environment. And it seems to me that `bower link` is a more robust approach, since there's no way to know what bower might otherwise do to the contents of that folder on update.